packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -53,9 +54,34 @@ const resolveSideEffects = () => ({
 	head: Helmet.rewind(),
 	redirect: Redirect.rewind(),
 	forbidden: Forbidden.rewind(),
-	notFound: NotFound.rewind(),
+	notFound: NotFound.rewind()
 });
 
+/**
+ * Using the User agent string + an external library called Mobile-Device we detect if the user
+ * is attempting to render the page on the server using a mobile, tablet, or desktop device
+ * @param  {String} uas User agent string detected from headers
+ * @return {Object}     Object of which category the devices falls within
+ */
+const getDeviceType = uas => {
```

Comment:
I'm hoping that Flowtype will error on the missing type definitions for this function, although I just realized that Flow type checking isn't part of the PR build currently - try running `yarn flow` from inside `packages/mwp-core/`